### PR TITLE
test pull request

### PR DESCRIPTION
**Pull request by chakkriii**

### CODE REVIEW

File: set1/Q1.cpp.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Question link : https://practice.geeksforgeeks.org/problems/k-largest-elements/0
+#include <iostream>
+#include<algorithm>
+#include<vector>
+using namespace std;
+vector<int> arr;
+
+int main()
+{
+	int t, n, k;
```
Comment (ktapasya) [inline set1/Q1.cpp.txt:10]: Please name the variables properly.

File: set1/Q1.cpp.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	//freopen("Testcase.txt", "r", stdin);
+	//freopen("output.txt", "w", stdout);
+	cin >> t;
+	while (t-- > 0) {
+		cin >> n >> k;
+		for (int i = 0; i < n; i++) {
+			cin >> arr[i];
+		}
+		make_heap(arr.begin(), arr.end());
```
Comment (ktapasya) [inline set1/Q1.cpp.txt:19]: Separate your algorithm logic into different function.

File: set1/Q1.cpp.txt [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+		for (int i = 0; i < k; i++) {
+			cout << arr[0] << " ";
+			pop_heap(arr.begin(), arr.end());
+		}
+	}
+	return 0;
+}
+
```
Comment (ktapasya) [inline set1/Q1.cpp.txt:27]: Also it would be better if you explain a little bit and mention the complexity of the code in comments.

**Review by ktapasya — CHANGES_REQUESTED** [review]
**Test Review**
File extension is txt.

File: set1/Q1.cpp.txt (hunk 1 above)
Comment (chakkriii) [reply set1/Q1.cpp.txt:10]: The variable names are according to question stated in the comment section

Comment (ktapasya) [reply set1/Q1.cpp.txt:10]: It is T, N and C[i] then. 😂

**Pull request merged**